kubernetes/redpanda/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Default values for redpanda.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: vectorized/redpanda
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+redpanda:
+  advertised_rpc_api:
+    port: 33145
+  advertised_kafka_api:
+    port: 9092
+  admin:
+    port: 9644
+  smp: 1
+
+volumeMount:
+  name: config
+  mountPath: /tmp/config 
+  base:
+    name: base-config
+    mountPath: /tmp/base-config
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: 
+  requests:
+    cpu: 1   # 2
+    memory: 3Gi
+  limits:
+    cpu: 1   # 2
+    memory: 6Gi
+
+nodeSelector: {}
```

Comment:
Seems unused, should be added to podSpec.

Reply:
Done